packages/components/src/FormWizard/FormFieldRenderer.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import React from 'react'
+
+type ComponentsMap = Record<string, React.ComponentType<any>>
+
+export type Field<C extends ComponentsMap> = {
```

Comment:
Could benefit from comment, maybe a good practice for components and types alike

Reply:
👍 Will do. Will clean these up as well, as I think they don't work as intended in narrowing the type at the moment.